owncloudApp/src/main/res/layout/create_shortcut_dialog.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical"
+    android:padding="@dimen/standard_half_margin">
+
+    <TextView
+        android:id="@+id/create_shortcut_dialog_title"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:padding="@dimen/standard_half_padding"
+        android:text="@string/create_shortcut_dialog_title"
+        android:textAppearance="@android:style/TextAppearance.Material.DialogWindowTitle" />
+
+        <LinearLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:orientation="vertical">
+
+            <RelativeLayout
```

Comment:
Well, `RelativeLayout` is not recommended anymore, it's better to use `ConstraintLayout`. But you can leave it like this for this case, just take it into account for the future